src/components/intent.ts

```diff
@@ -575,6 +575,17 @@ export class Intent {
         return this.client.setAvatarUrl(url);
     }
 
+    public async setRoomUserDisplayName(roomId: string, name: string) {
+        // FIXME: the content should be based on the current membership event
+        // (do we have this cached somewhere?).  Also, do nothing if the given
```

Comment:
`this.opts.backingStore.getMembership(roomId, userId)` will get you the membership key